themes/default/content/learn/building-with-pulumi/stack-outputs/index.md

```diff
@@ -56,6 +56,19 @@ Note that you'll want to insert this just before the last `return nil` statement
 
 {{% /choosable %}}
 
+{{% choosable language csharp %}}
+
+```csharp
+return new Dictionary<string, object?>
+{
+    ["url"] = pulumi.Output.Format($"http://localhost:{frontendPort}")
```

Comment:
This should be just `Output.Format($"http://localhost:{frontendPort}")`, right? the dotnet SDK doesn't have a lower-case `pulumi` namespace

Reply:
herp yes, too much copy paste